spec/prog/vm/github_runner_spec.rb

```diff
@@ -18,7 +18,7 @@
   }
 
   let(:vm) {
-    Vm.new(family: "standard", cores: 1, name: "dummy-vm", location: "hetzner-hel1").tap {
+    Vm.new(family: "standard", cores: 1, name: "dummy-vm", location: "github-runners").tap {
```

Comment:
why do we change location to github-runners here?

Reply:
Because it's `github_runner_spec.rb` and the location of the billing rate was `github-runners` initial PR. It's `global` now, and not needed it. But I think it's good to keep it that way to be close to real examples.